hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BootstrapConfig.java

```diff
@@ -9,7 +9,7 @@
 
 @ConfigData("bootstrap")
 public record BootstrapConfig(
-        @ConfigProperty(value = "feeSchedulesJson.resource", defaultValue = "/genesis/feeSchedules.json")
+        @ConfigProperty(value = "feeSchedulesJson.resource", defaultValue = "genesis/feeSchedules.json")
```

Comment:
Reverts change from #18922 - for consistency the `/` is prepended in `loadResourceInPackage(...)`.